src/screens/Search/SearchFilter.js

```diff
@@ -154,7 +154,7 @@ class SearchFilter extends Component {
     const { key } = item;
 
     if (key === 'length') {
-      return this.renderCustomItem('Length', 'length');
+      return this.renderCustomItem('Epsiodes', 'length');
```

Comment:
This wouldn't work since we also have manga

Reply:
Plus it's misspelled